dp_check/dp_check.go

```diff
@@ -209,7 +377,7 @@ The most likely reason for this is that %s is not a DirectPath-enabled service`,
 		if len(balancerAddr) == 0 {
 			return fmt.Errorf("Skipping TCP connectivity to load balancers because load balancer DNS resolution failed")
 		}
-		infoLog.Printf("Check TCP connectivity to LB's with:|net.DialTimeout(\"tcp\", \"%v\", time.Second*5)|...\n", balancerAddr)
+		infoLog.Printf("Check TCP connectivity to LB's with:|net.DialTimeout(\"tcp\", \"%v\", time.Second*5)|...", balancerAddr)
```

Comment:
here and elsewhere, were newlines intentionally removed at the end of logs? Are they still getting logged?

Reply:
newline is automatically appended in log package.